packages/gatsby-theme-newrelic/src/components/SEO.js

```diff
@@ -94,6 +94,20 @@ const SEO = ({ title, location, type, children }) => {
           />
         );
       })}
+
+      {/* Android */}
+      <meta name="theme-color" content="--primary-background-color" />
```

Comment:
`--primary-background-color` is white in light mode and black in dark mode, so i don't think this color is coming through at all. will this still work if we remove the content property entirely? it feels like its falling back to the header color which is good but if someone looks at this in the future it doesn't make sense

Reply:
better yet, if the different envs don't fallback the same way we might just want to hardcode the header color. not the best, but safe #293338

Reply:
@LizBaker, I got this update and it should be good to go if y'all like it!